packages/bitwuzla-bin/bitwuzla-bin.0.0.0/opam

```diff
@@ -0,0 +1,44 @@
+opam-version: "2.0"
+synopsis: "Bitwuzla SMT solver executable"
+description: """
+
+Standalone installer for the SMT solver Bitwuzla.
+
+Bitwuzla is a Satisfiability Modulo Theories (SMT) solver for the theories of fixed-size bit-vectors, arrays and uninterpreted functions and their combinations. Its name is derived from an Austrian dialect expression that can be translated as “someone who tinkers with bits”."""
+maintainer: ["Frédéric Recoules <frederic.recoules@cea.fr>"]
+authors: ["Frédéric Recoules"]
+license: "MIT"
+tags: ["SMT solver" "SMT-COMP 2020" "QF_AUFBVFP"]
+homepage: "https://bitwuzla.github.io"
+bug-reports: "https://github.com/bitwuzla/ocaml-bitwuzla/issues"
+depends: [
+  "dune" {>= "2.6"}
+  "conf-git" {build}
+  "conf-cmake" {build}
+  "conf-gcc" {build}
+  "conf-g++" {build}
+  "conf-gmp" {build}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+  ]
```

Comment:
why has the tests and doc been removed from there? It seem to be only a matter of adding a missing test dependency

Reply:
Because this package is just a convenient way to install bitwuzla in `bin`. There is no test associated to it and so, no reason to force to install useless dependencies.